backend/src/main/java/com/ody/mate/dto/request/MateStatusRequest.java

```diff
@@ -0,0 +1,20 @@
+package com.ody.mate.dto.request;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.annotation.Nullable;
+
+public record MateStatusRequest(
```

Comment:
[제안]
`MateEtaRequest` 로 바꾸는건 어떤가요?

```suggestion
public record MateEtaRequest(
```

Reply:
처음에는 api 응답에 있는 데이터명 그대로 따라가다가 `MateStatusRequest`로 네이밍 하다
참여자별 상태를 갖는 Enum은 `EtaStatus`로 네이밍하게 되었는데
개인적으로는 Eta가 현재 응답 DTO에 대한 이름을 잘 명시된 것 같아서 
Dto명도 `MateEtaRequest`로 네이밍하고 api uri도 수정되면

Reply:
dto 이름으로 `MateEtaRequest`를 사용하는 거 좋아요
uri는 `/v1/meetings/{meetingId}/mates/etas` 사용하는 것 어떤가요?

Reply:
`/v1/meetings/{meetingId}/mates/etas` 맘에 드네요 ! 👍🏻

DTO명과 URI를 변경하는데 다들 동의하는 것 같아 DTO 명 `MateEtaxxx`로 변경과 더불어 반영했습니다 😄